imap_processing/swe/l1b/swe_l1b_science.py

```diff
@@ -40,7 +40,7 @@
 }
 
 
-def read_lookup_table(table_index_value: int):
+def read_lookup_table(table_index_value: int) -> list:
```

Comment:
```suggestion
def read_lookup_table(table_index_value: int) -> Any:
```

I think this will solve the errors in this function?